### PR TITLE
Removed problematic toast (was not specified in Figma for this view)

### LINKED ISSUE #617: Timetable Feature: Fix an action of tapped favorite button
## Overview

Currently, timetable cell's favorite button tapped action isn't correct.

## Goal

- change action when tapped favorite button action to switching heart icon.

## Movie

<video src="https://github.com/user-attachments/assets/35f3bdb6-0c7b-42c9-909b-be1a3fb9d392" />

### PR DESCRIPTION
**Pull request by charles-b-stb**
## Issue
- close #617

## Overview (Required)
- Toast message removed from timetable list view. The toast conflicted with the floating tab bar and was not specified in Figma (unlike the detail view) so it was removed to match spec.

## Links
- 

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
 | 

## Movie (Optional)
Before | After
:--: | :--:
<video src="" width="300" > | <video src="" width="300" >

### CODE REVIEW

**Comment by shin-usu** [comment]
Probably, you can remove toast value from state.
Also, AddFavorite string can be remove, I think.

**Comment by charles-b-stb** [comment]
@shin-usu Fixed as suggested. Thanks!

**Comment by shin-usu** [comment]
It seems like build error is occurring 👀

**Comment by charles-b-stb** [comment]
@shin-usu Sorry about that. This should fix that build issue. Tested again locally and it seems to be fine.

**Review by shin-usu — COMMENTED** [review]
@charles-b-stb 
Thank you for your commit! 
I left some comments. Please check and response🙏🏻

File: app-ios/Sources/TimetableFeature/TimetableReducer.swift [hunk 1]
```diff
@@ -104,10 +103,8 @@ public struct TimetableReducer : Sendable{
                         return item.isFavorited
                     }))
                 }
-            case let .favoriteResponse(.success(isFavorited)):
-                if !isFavorited {
-                    state.toast = .init(text: String(localized: "AddFavorite", bundle: .module))
-                }
+            case .favoriteResponse(.success(_)):
+                //No response here on purpose. Toasts look bad here.
```
[the remark this answers is not in the document]
Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableReducer.swift]: Fixed

File: app-ios/Sources/TimetableFeature/TimetableReducer.swift [hunk 2]
```diff
@@ -103,7 +103,7 @@ public struct TimetableReducer : Sendable{
                         return item.isFavorited
                     }))
                 }
-            case let .favoriteResponse(.success(isFavorited)):
+            case .favoriteResponse(.success(_)):
```
[the remark this answers is not in the document]
Comment (charles-b-stb) [reply app-ios/Sources/TimetableFeature/TimetableReducer.swift]: fixed

**Review by shin-usu — APPROVED** [review]
LGTM🎉🎉 Thanks!!

**Pull request merged**